src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -21,7 +21,9 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class DecryptTests
     {
-        [Fact]
+        public static bool IsNet462OrNewer() => (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());
```

Comment:
Also applies to the other two files.  Every failing test used one of the RSATransferX certs, and the RSATransferCapiX certs passed.